apps/funding-platform/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "rpch-funding-platform",
+  "version": "0.0.0",
+  "private": true,
+  "license": "MIT",
+  "main": "./build/index.js",
+  "types": "./build/index.d.ts",
+  "scripts": {
+    "build": "rimraf build && tsup",
+    "dev": "yarn build --watch",
+    "format": "prettier --check \"src/**/*.{ts,md}\"",
+    "lint": "TIMING=1 eslint \"src/**/*.ts*\"",
+    "test": "jest --coverage",
+    "start": "node build/index.js"
+  },
+  "devDependencies": {
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.7",
+    "@nomiclabs/hardhat-ethers": "^2.2.1",
+    "@types/express": "^4.17.14",
+    "@types/supertest": "^2.0.12",
+    "dotenv": "^16.0.3",
```

Comment:
`dotenv` here is a dev dependency, but I see it included in one of our runtime files